Decompiler/Program.cs

```diff
@@ -596,19 +610,25 @@ private static void ParseVPK(string path)
 
             sw.Stop();
 
-            Console.WriteLine("Processed in {0}ms", sw.ElapsedMilliseconds);
+            if (!Options.Silent) Console.WriteLine("Processed in {0}ms", sw.ElapsedMilliseconds);
         }
 
         private static void DumpVPK(Package package, string type, string newType)
         {
+            if (!string.IsNullOrEmpty(Options.FilterExt) && !Options.FilterExt.Contains(type))
+            {
+                return;
+            }
+
             if (!package.Entries.ContainsKey(type))
             {
-                Console.WriteLine("There are no files of type \"{0}\".", type);
+                if (!Options.Silent) Console.WriteLine("There are no files of type \"{0}\".", type);
 
                 return;
             }
 
             var entries = package.Entries[type];
+            uint oldCrc32;            
```

Comment:
Won't compile on Visual Studio Community Edition 2015, will remove it next edit.